src/microjson/tile.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional, Union, Dict
+from pydantic import BaseModel, AnyUrl, conlist, RootModel
+from pathlib import Path
+
+
+class TileLayer(BaseModel):
+    id: str
+    fields: Union[None, Dict[str, str], Dict[str, Dict]]
+    minzoom: Optional[int] = None
```

Comment:
change minzoom and maxzoom to Camel case

Reply:
They are harmonized with TileJSON terminology.